lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -315,10 +315,15 @@ module.exports = [
         },
         'layout-shift-elements': {
           score: null,
-          scoreDisplayMode: 'notApplicable',
-          details: {
-            items: [],
-          },
+          // If nodes were evicted, then `details.items === []`, and we mark things as notApplicable. #10877
+
+          // Our test page tries to _force_ an eviction (#11426)
+          // But we've seen cases where there is no eviction. This is fine, actually. The user
+          // ends up seeing a useful result. (and item.details is populated!)
+          // This case (just like the normal here-are-your-cls-elements case) is marked `informative`.
+
+          // That leaves us mostly asserting that this audit is error-free.
+          scoreDisplayMode: /(notApplicable|informative)/,
```

Comment:
Does this make the test mostly useless since it's supposed to be a test of a page with eviction? I believe `largest-contentful-paint-element` only has `scoreDisplayMode: /(notApplicable|informative)/` because it allows for testing before and after m88, not because of a flake.

Is this still flaky in CI or maybe it could be "fixed" in `smokerider-entry` instead?

Reply:
My intention for `notApplicable` [originally](https://github.com/GoogleChrome/lighthouse/commit/91b4461c214c0e05d318ec96f6585dcca52a51cc) was so that we receive a heads up when the test is no longer testing what we want it to because Chrome changed or something, but the primary goal was indeed just being "error-free". 

If it's just LR that has this problem, +1 to a smokerider specific fix. If it's everywhere, then we should take a look at the test and make it more resilient (moar nodes perhaps? :D)

Reply:
IMO this smoketest is evaluating if the fixture page is successfully causing an eviction.
it's not testing lighthouse's correct behavior.

like... is it a problem if we have items?

Reply:
> IMO this smoketest is evaluating if the fixture page is successfully causing an eviction.

That feels a little narrow to me ;) It's asserting a combination of two things:

- The fixture page caused the situation we care about testing
- The audit did not throw an exception in that situation

> like... is it a problem if we have items?

No of course not :) But if this test never causes an eviction then there's no point in it existing and we have no coverage of what happens when elements do get evicted.